### DISCUSSION TITLE
Make it so the bar only shows on 1 monitor

### DISCUSSION THREAD

**Discussion by Tnixc** [FelixKratz/SketchyBar#260] [Q&A] [answered]
Having `sketchybar --bar associated_display=1\` does not work. https://github.com/FelixKratz/SketchyBar/discussions/213 Is somewhat similar.

**Comment by Tnixc** [comment, marked as answer]
Nevermind, didn't read the docs properly. Close this

**Reply by umtiagopereira** [reply]
Hi. Were you able to do this?

**Reply by mingsheng13** [reply]
Use sketchybar --bar display=main
